renovate.json

```diff
@@ -9,7 +9,7 @@
   ],
   "bazelisk": {
     "fileMatch": [
-      "(^|/)examples/gem/\\.bazelversion$"
+      "\\.bazelversion"
```

Comment:
I swear this is a last time I try to keep all `.bazelversion` files in sync.

Reply:
I usually check in symlinks for that purpose

Reply:
Good idea!